source/_integrations/motion_blinds.markdown

```diff
@@ -40,6 +40,10 @@ Please note that "-" characters need to be included in the key when providing it
 <img src='/images/integrations/motion_blinds/Motion_App__get_key_2.jpg' />
 </p>
 
+Note that it has been reported that the official Bloc Blinds app doesn't seem to hand out the API key on Android, it does seem to provide the API key on the iOS version of the official Bloc Blinds app.
+You can also retrieve the API key by using a packet capture app on a mobile phone and "sniffing" the packets of the Bloc Blinds app.
+Search for a packet containing `accessToken=`, the first 16 characters of the value (including -) is the API key.
```

Comment:
Is that something we should advise to our end-users?

Reply:
Well I don't know.
On the one hand it is not nice for the end-users, on the other hand someone that only has android phones and no acces to an iOS phone and uses Bloc Blinds will otherwise not be able to setup the component (Or they have to find some other way to get the accesToken, maybe contact Bloc Blinds or something...)

Personly I don't have Bloc Blinds, so I am also not able to check or experiment.

I am fine whith removing the part about packet capture, what do you think @frenck

Reply:
I think the average user has no clue what you have written here 🤷

Reply:
I have removed the part about the packet capture